fastcrypto-zkp/src/bn254/poseidon/mod.rs

```diff
@@ -223,21 +254,20 @@ mod test {
         )
         .unwrap();
 
-        let hash = poseidon
-            .hash(vec![
-                jwt_sha2_hash_0,
-                jwt_sha2_hash_1,
-                masked_content_hash,
-                payload_start_index,
-                payload_len,
-                eph_public_key_0,
-                eph_public_key_1,
-                max_epoch,
-                num_sha2_blocks,
-                key_claim_name_f,
-                addr_seed,
-            ])
-            .unwrap();
+        let hash = hash(vec![
```

Comment:
is this a zkLogin input test? if yes why it's not in the zkLogin test file? is it because it happens to have the number of inputs we expect for this test?

Reply:
Good point. I've moved the test.